client/src/modules/Job/pages/NoAuthLanding/components/JobSeekerInfo.tsx

```diff
@@ -0,0 +1,126 @@
+import { Button } from 'antd';
+import { COLORS } from 'modules/Job/theme/colors';
+
+export const JobSeekerInfo = () => {
+  return (
+    <>
+      <div className="wrapper">
```

Comment:
Same as above regarding usage of Antd components instead of raw html and styles

Reply:
the same as above